inc/languages/english/global.lang.php

```diff
@@ -328,8 +328,9 @@
 $l['unapproved_attachments'] = "{1} unapproved attachments";
 $l['unread_report'] = "1 unread report";
 $l['unread_reports'] = "{1} unread reports";
-$l['pending_joinrequest'] = "Group Leader Notice: You have 1 pending group membership join request.";
-$l['pending_joinrequests'] = "Group Leader Notice: You have {1} pending group membership join requests.";
+$l['groupleader_notice'] = "Group Leader Notice:";
```

Comment:
What about "Pending Group Leader Action(s)"?

Reply:
How many _actions_ are there for group leaders?